Apps/Src/SendTritium.c

```diff
@@ -32,7 +32,7 @@
 #define MIN_CRUISE_VELOCITY mpsToRpm(20.0f) // rpm
 #define MAX_GEARSWITCH_VELOCITY mpsToRpm(8.0f) // rpm
 
-#define BRAKE_PEDAL_THRESHOLD 15  // percent
+#define BRAKE_PEDAL_THRESHOLD 50  // percent
```

Comment:
what do these macros mean? None of them have descriptions

Reply:
They handle what threshold do we consider the reading of the pedal to be to be considered on. This documentation should happen when we go through and document the code more thoroughly